.travis/BuildLinux.sh

```diff
@@ -57,15 +57,15 @@ if [ -z "${RUN_CLANG_TIDY}" ] \
     && [ -z "${RUN_IWYU}" ] \
     && [ -z "${BUILD_DOC}" ]; then
     if [[ ${TRAVIS_BUILD_STAGE_NAME} = "Build libraries" ]]; then
-        make libs test-libs-stage1 -j2
+        time make libs test-libs-stage1 -j2
```

Comment:
Do you want to keep the timing records? I personally have no objection to them :)

Reply:
yeah, I don't see how they hurt...